#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_cosmetics.dpatch by Daniel Kahn Gillmor <dkg@fifthhorseman.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Cosmetic fixes which could be applied upstream

@DPATCH@
diff -urNad redir-2.2.1~/redir.man redir-2.2.1/redir.man
--- redir-2.2.1~/redir.man	2005-10-22 21:41:51.284918168 -0400
+++ redir-2.2.1/redir.man	2005-10-22 21:42:42.115190792 -0400
@@ -73,7 +73,7 @@
 Specify program name to be used for TCP wrapper checks and syslog logging.
 .TP
 .B --timeout
-Timeout and close the connection after n seconds on inactivity.
+Timeout and close the connection after n seconds of inactivity.
 .TP
 .B \--syslog
 Log information to syslog.
@@ -90,7 +90,7 @@
 undesirable. 
 .TP
 .B \--transproxy
-On a linux system with transparany proxying enables, causes redir to
+On a linux system with transparent proxying enabled, causes redir to
 make connections appear as if they had come from their true origin.
 (see transproxy.txt in the source archive)
 .TP
